lib/health_cards/covid_health_card.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module HealthCards
+  # Implements HealthCard for use with COVID Vaccination IG
+  class COVIDHealthCard < HealthCards::HealthCard
+    fhir_version '4.0.1'
+    additional_types 'https://healthwallet.cards#covid19'
+
+    allow FHIR::Patient, %w[name birthDate]
+    allow FHIR::Immunization, %w[status vaccineCode patient occurrenceDateTime]
```

Comment:
This looks really nice, but I worry about how complicated this might have to become.

Reply:
Yeah, I spoke to @radamson about this. We agreed that we'll deal with that when we need to. I think any crazy requirements we run into in the future would require an equally crazy implementation anyways.